packages/peer-exchange/src/waku_peer_exchange_discovery.ts

```diff
@@ -3,16 +3,20 @@ import {
   PeerDiscoveryEvents,
   symbol,
 } from "@libp2p/interface-peer-discovery";
+import { PeerId } from "@libp2p/interface-peer-id";
 import { PeerInfo } from "@libp2p/interface-peer-info";
 import { PeerProtocolsChangeData } from "@libp2p/interface-peer-store";
 import { EventEmitter } from "@libp2p/interfaces/events";
 import { PeerExchangeComponents } from "@waku/interfaces";
 import debug from "debug";
 
+import { WakuPeerExchange } from "./waku_peer_exchange";
```

Comment:
```suggestion
import { WakuPeerExchange } from "./waku_peer_exchange.js";
```

`tsc` should fail without the extension. Not sure why it doesn't? is this file included in the compilation?